Jenkinsfile

```diff
@@ -4,6 +4,47 @@ pipeline {
   agent none
 
   stages {
+    stage('Update changelog') {
+      agent any
+
+      when {
+        anyOf {
+          branch 'master'
+          expression { BRANCH_NAME ==~ /^\d+(.\d+)*$/ }
+        }
+      }
+
+      steps {
+        sh "docker pull jsternberg/changelog"
+        withDockerContainer(image: "jsternberg/changelog") {
+          withCredentials(
+            [[$class: "UsernamePasswordMultiBinding",
+              credentialsId: "hercules-username-password",
+              usernameVariable: "GITHUB_USER",
+              passwordVariable: "GITHUB_TOKEN"]]) {
+            script {
+              if (env.GIT_PREVIOUS_SUCCESSFUL_COMMIT) {
+                sh "git changelog ${env.GIT_PREVIOUS_SUCCESSFUL_COMMIT}"
```

Comment:
where is this env var set?  is it in a different PR?  

the jenkins code looks good to me.  I'm a bit lost on what happens when it runs but it's because I lack context on the 'git changelog'  command.